lib/modules/platform/gitea/index.md

```diff
@@ -20,6 +18,7 @@ Either the account should have full name and email address set to allow Renovate
 
 - **Adding reviewers to PRs not supported**: Gitea versions older than v1.14.0 do not have the required API.
 - **Setting `platformAutomerge` to use platform-native automerge for PRs not supported**: Gitea versions older than v1.17.0 do not have the required API.
+- **Git upload filters**: On Gitea versions older than v1.17.0 you need to manually enable [clone filters](https://docs.gitea.io/en-us/clone-filters/).
```

Comment:
```suggestion
- **Git upload filters**: On Gitea versions older than v1.16.0 you need to manually enable [clone filters](https://docs.gitea.io/en-us/clone-filters/).
```
Was backported to v1.16.0.

https://github.com/go-gitea/gitea/blob/main/CHANGELOG.md#1160---2022-01-30